core/src/main/java/io/seata/core/rpc/netty/AbstractRpcRemoting.java

```diff
@@ -293,17 +308,19 @@ protected void sendRequest(Channel channel, Object msg) {
     /**
      * Send response.
      *
-     * @param msgId   the msg id
+     * @param request  the msg id
      * @param channel the channel
      * @param msg     the msg
      */
-    protected void sendResponse(long msgId, Channel channel, Object msg) {
+    protected void sendResponse(RpcMessage request, Channel channel, Object msg) {
```

Comment:
just use request.getId why wrap as RpcMessage.   if use RpcMessage ,it is best to wrap msg in the body